commit 728973c9711639447bb908d89b2f46c89228753f
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Thu Feb 16 16:21:25 2023 -0500
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Feb 16 16:21:25 2023 -0500

    FL server deploy integrate with cellnet (#1376)
    
    * FL server integrate with FCI cellnet.
    
    * Codestyle reformat.
    
    * fed_server_test.py integrate with Cellnet.
    
    * Fixes according to PR reviews.
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 280bc8e8..5e2e407a 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -837,7 +837,7 @@ def server_shutdown(server, touch_file):
         os.utime(touch_file, None)
 
     try:
-        server.fl_shutdown()
+        server.fl_shutdown(shutdown_period=30.0)
         server.admin_server.stop()
         time.sleep(3.0)
     finally:
